fsspec/implementations/arrow.py

```diff
@@ -294,4 +296,6 @@ def _get_kwargs_from_urls(path):
             out["user"] = ops["username"]
         if ops.get("port", None):
             out["port"] = ops["port"]
+        if ops.get("replication", None):
+            out["replication"] = ops["replication"]
```

Comment:
How would you specify the replication in a URL, what would that mean?

Reply:
refer to this [documentation](https://arrow.apache.org/docs/python/generated/pyarrow.fs.HadoopFileSystem.html#pyarrow.fs.HadoopFileSystem.from_uri) some thing like this:

`hdfs://localhost:8020/?user=test&replication=1`

Reply:
If you put that URL through infer_storage_options, "user" and "replication" and bundled in the `query` field.

Reply:
This is still outstanding. In effect, I don't think this has any effect, since `ops` will never include they key - but it would be nice to write a test for it.

Reply:
I apologize for the delay. I didn't understand the behavior of the infer_storage_options function correctly. I have now fixed that and I have written a test for it.